infra/gcp/ensure-release-prod-projects.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to ensure Release Engineering subproject owners have the
+# appropriate access to SIG Release prod GCP projects (k8s-staging-kubernetes).
+
+# TODO(justaugustus): Move k8s-staging-kubernetes access grants to
+#                     ensure-release-projects.sh once we've completed testing
+#                     and documenting the staging/image promotion flow.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+. "${SCRIPT_DIR}/lib.sh"
+
+function usage() {
+    echo "usage: $0 [project...]" > /dev/stderr
+    echo "example:" > /dev/stderr
+    echo "  $0 # do all release projects" > /dev/stderr
+    echo "  $0 k8s-staging-release-test # just do one" > /dev/stderr
```

Comment:
Should this be `k8s-staging-kubernetes` ?

Reply:
@listx -- It was just the usage example, but I've updated for consistency.